### PR TITLE
Add French translation

### PR DESCRIPTION
**Pull request by Aethys256**
French translations 👍

### CODE REVIEW

File: languages/fr/modules.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ * @package    TableLookupWizard
+ * @link       http://www.terminal42.ch
+ * @license    http://opensource.org/licenses/lgpl-3.0.html LGPL
+ */
+
+
+/**
+ * Extension folder
+ */
+$GLOBALS['TL_LANG']['MOD']['tablelookupwizard'] = array('Assistant de recherche de table');
```
Comment (Aethys256) [inline languages/fr/modules.php]: I do have a doubt for this one.
Not sure if you want to translate 'table' like a 'table' or like an 'entry'/'record'.

Comment (Toflar) [reply languages/fr/modules.php]: It's only shown in the system settings anyway. Nothing an end user will ever see so it may remain a technical term :)

**Comment by Toflar** [comment]
Thank you, it's ready in the `develop` branch. Also there's a new feature to limit the sql results. See https://github.com/terminal42/contao-tablelookupwizard/issues/12. As soon as I get the OK, I'll release :)

**Comment by Toflar** [comment]
Hey @Aethys256! We've just added another feature to limit the number of results. If there are more than can be displayed, we show the hint "More results have been found than can be displayed. Please refine your search criteria.". See https://github.com/terminal42/contao-tablelookupwizard/commit/b23086bacc2efc4f45638c5993adbbf02e4f6ee1.

Would you like to translate that into French too? I can then release version 3.3 😄

**Comment by Aethys256** [comment]
No problem, I'll do tonight/tomorrow and try it out the limit feature aswell 👍

**Comment by Toflar** [comment]
You don't have to PR, just translate the sentence, I'll commit it, no prob :)

**Comment by Aethys256** [comment]
Oh ok :

> More results have been found than can be displayed. Please refine your search criteria.

> Seule une partie des résultats est affichée. Veuillez affiner vos critères de recherche.

In the same time, I think this sentence is english would be more appropriate:

> Only a part of the results is displayed. Please refine your search criteria.

**Comment by Toflar** [comment]
Thank you, I have adjusted everything and released version 3.3.0, enjoy 😄

**Pull request merged**